ftw/file/content/file.py

```diff
@@ -28,8 +29,11 @@
 import logging
 
 
+class FileImageField(FileField, ImageFieldMixin):
```

Comment:
``FileField`` is in ``ftw.file.field``, I'd not subclass it but add the image mixin there